### PR TITLE
Intune install method to use Settings Catalog

### PR DESCRIPTION
**Pull request by joshuahipple**
Settings Catalog can be used now to manage the Chrome extensions. Chrome ADMX ingestion (and thus custom OMA-URI) is no longer needed.

### CODE REVIEW

**Comment by joshuahipple** [comment]
@microsoft-github-policy-service agree

File: microsoft-365/compliance/dlp-chrome-get-started.md [hunk 1]
```diff
@@ -144,23 +142,23 @@ Before adding the extension to the list of force-installed extensions, it is imp
...
 
-9. Select **Add** on the **Configuration settings** tab.
-
-10. Enter the following policy information.
+9. Select **Add settings** on the **Configuration settings** tab.
+10. Select "Administrative Templates \ Google \ Google Chrome \ Extensions"
+11. Select "Configure the list of force-installed apps and extensions"
+12. Change the toggle to enabled
+13. Enter the following value for the Extension/App IDs and update URL:
+     echcggldkblhodogklpincgchnpgcdco;https://clients2.google.com/service/update2/crx
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-chrome-get-started.md]:
```suggestion
10. Select **Administrative Templates** > **Google** > **Google Chrome** > **Extensions**.
11. Select **Configure the list of force-installed apps and extensions**.
12. Change the toggle to **Enabled**.
13. Enter the following value for the extensions and app IDs and update URL:
     `echcggldkblhodogklpincgchnpgcdco;https://clients2.google.com/service/update2/crx`.
```

File: microsoft-365/compliance/dlp-chrome-get-started.md [hunk 2]
```diff
@@ -132,9 +132,7 @@ Use this setup method for organization-wide deployments.
 
 #### Microsoft Intune Force Install Steps
 
-Before adding the extension to the list of force-installed extensions, it is important to ingest the Chrome ADMX. Steps for this process in Microsoft Intune are documented by Google: [Manage Chrome Browser with Microsoft Intune - Google Chrome Enterprise Help](https://support.google.com/chrome/a/answer/9102677?hl=en#zippy=%2Cstep-ingest-the-chrome-admx-file-into-intune).
-
- After ingesting the ADMX, the steps below can be followed to create a configuration profile for this extension.
+Settings Catalog can be used now to manage the Chrome extensions.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-chrome-get-started.md]:
```suggestion
Using the settings catalog, follow these steps to manage Chrome extensions:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @joshuahipple, kindly commit the suggested changes if you agree with them. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by chrfox** [comment]
confirming with PM

**Pull request merged**